styled-switch-view/build.gradle

```diff
@@ -45,7 +45,7 @@ dependencies {
     api rootProject.ext.dep.kotlinStdlibJdk7
 
     // Project's core library
-    api project(path: ':styled-core')
+    compileOnly project(path: ':styled-core')
```

Comment:
Why do we have to import this both at project-level and then in the module as well?

Reply:
@boyw165 compileOnly means that third-party would be forced to also import our core lib, correct??

Reply:
@xuhaibahmad : Previously, the extensions are not declared as `inline`, so they are compiled to instructions in the `styled-core` module. Using `api` or `implementation` both requires the third-party to include the `styled-core` module.

@kevinmmarlow : `compileOnly` hides the dependency from third-party, which means this dependency is only used in assembling the byte-code. Because of using `inline` extensions, we guarantee the instruction get inlined and then with `compileOnly` we hide the dependency from third-party.